content/en/apps/tutorials/application-settings.md

```diff
@@ -0,0 +1,184 @@
+---
+title: "CHT Application Settings"
+linkTitle: Application Settings
+weight: 1
+description: >
+  Managing CHT application settings
+relatedContent: >
+  
+---
+
+## Purpose of the Tutorial
+
+This tutorial will take you through how to manage the CHT application settings, including; setting user roles and permissions, enabling and disabling transitions, configuring contact hierarchy and configuring replication.
+
+## Brief Overview of Key Concepts
+
+The settings which control CHT apps are defined in the *app_settings.json* file, and stored in the settings doc in the database.
+
+*Permissions* are settings that control access to specific app features and functionality.
+
+*Roles* define permissions for users to perform access a group of app features and functionality.
+
+*[Replication depth](https://docs.communityhealthtoolkit.org/apps/guides/performance/replication/#depth)* is the number of levels a specific user role is able to replicate on their application.
```

Comment:
This is probably a foreign concept to many of our users. Would be good to add a little extra description. Maybe something like:
Replication is when users download a copy of the data on to their device. Replication depth refers to the number of levels within a hierarchy a specific user role is able to replicate.

Reply:
Updated.